src/dw_apb_uart.cpp

```diff
@@ -181,20 +183,22 @@ void dw_apb_uart_write(void *opaque, uint32_t offset, uint32_t val, int size_log
 
     assert(offset % 4 == 0 && offset / 4 < 64);
 
-    if (reg_names[offset / 4] && size_log2 == 2)
-        DEBUG("dw_apb_uart_write(0x%02x \"%s\", %d)\n", offset, reg_names[offset / 4], val);
+    if (1 && offset == 0)
```

Comment:
This is a bit strange `if (true and offset==0)` should be the same as `if (offset==0)`